statistic-labels/index.html

```diff
@@ -338,6 +339,11 @@ <h3 id="printBasinTypeHeader">Basin Characteristic Types</h3>
 			  <select onChange="filterByStatGroup(this.value)" id="statTypeDropdown">
 				<option value="None">None</option>
 			  </select>
+			  <br>
+			  <label>Filter by Study Area:</label>
+			  <select onChange="filterByStudyArea(this.value)" id="studyAreaDropdown">
```

Comment:
Change the variables to be "Region" instead of "StudyArea" throughout? Unless there is reasoning for using StudyArea.

Reply:
I guess I was getting my wording from NSS, so maybe we need to think about changing that. Or at least ask why they are different.